charts/cluster-scanner/templates/NOTES.txt

```diff
@@ -1 +1,9 @@
-TODO
+The Cluster Scanner component is being executed.
+
+It consists of the Runtime Status Integrator and Image Sbom Extractor components.
+
+After successful installation you can access the Cluster Scanner features as follows:
+
+Log in to Sysdig Secure and check that the features are working as expected.
```

Comment:
@michele-mangili this will actually be rendered with values, as part of the chart (and could also get actual live data from the kubernetes cluster!!!)

see: https://helm.sh/docs/chart_template_guide/notes_files/

I'm saying this because we could generate the UI URL instead of telling how to reach it :)

Reply:
@FedericoFeresini I copy-pasted the message we currently display for the node analyzer https://github.com/sysdiglabs/charts/blob/006d2fc36547a0ab8c2f1d6a26bea75ac93651a8/charts/node-analyzer/templates/NOTES.txt 

In that file we have exactly the same statement: `Log in to Sysdig Secure and check that the features are working as expected.`. So I guess our customers expect something like that.